src/modules/dashboard/sagas/actions.ts

```diff
@@ -569,6 +573,126 @@ function* createMintTokensAction({
   }
 }
 
+function* createVersionUpgradeAction({
+  payload: { colonyAddress, colonyName, version, annotationMessage },
+  meta: { id: metaId, history },
+  meta,
+}: Action<ActionTypes.COLONY_ACTION_VERSION_UPGRADE>) {
+  let txChannel;
+  try {
+    const apolloClient = TEMP_getContext(ContextModule.ApolloClient);
+    const colonyManager = TEMP_getContext(ContextModule.ColonyManager);
+
+    const { version: newestVersion } = yield getNetworkContracts();
+    const currentVersion = parseInt(version, 10);
+    const nextVersion = currentVersion + 1;
+    if (nextVersion > parseInt(newestVersion, 10)) {
+      throw new Error('Colony has the newest version');
+    }
+
+    const supportAnnotation = currentVersion >= 5 && annotationMessage;
```

Comment:
Same here, please use the version constant from colonyJS

```suggestion
  const supportAnnotation = currentVersion >= ColonyVersion.CeruleanLightweightSpaceship && annotationMessage;
```